src/pages/home/report/ReportActionItemMessageEdit.js

```diff
@@ -118,16 +119,17 @@ function ReportActionItemMessageEdit(props) {
         });
 
         return () => {
-            // Skip if this is not the focused message so the other edit composer stays focused
-            if (!isFocusedRef.current) {
+            // Skip if this is not the focused message so the other edit composer stays focused.
+            // In small screen devices, when EmojiPicker is shown, the current edit message will lose focus, we need to check this case as well.
+            if (!isFocusedRef.current && !EmojiPickerAction.isActiveReportAction(props.action.reportActionID)) {
```

Comment:
This caused a bug where this condition returns false when composer is not focused but save button is pressed, since onFocus function is synchronous and takes some times to get focus. Thus causing edit input to be still open and main composer not to show: https://github.com/Expensify/App/issues/32218